test/test_KubeLibrary.py

```diff
@@ -326,9 +327,9 @@ def test_get_role_bindings_in_namespace(self, mock_lnp):
         role_bindings = kl.get_role_bindings_in_namespace('default')
         self.assertEqual(['read-pods'], [item for item in role_bindings])
 
-    @mock.patch('kubernetes.client.CoreV1Api.list_namespaced_config_map')
```

Comment:
you are removing test here, that is why coverage drops.

Reply:
Done.